articles/azure-functions/functions-develop-vs.md

```diff
@@ -102,7 +102,7 @@ In pre-compiled functions, the bindings used by the function are defined by appl
     } 
     ````
  
-    A binding-specific attribute is applied to each binding parameter supplied to the entry point method. The attribute takes the binding information as parameters. In the previous example, the first parameter has a **QueueTrigger** attribute applied, indicating queue triggered function. The queue name and connection string setting name are passed as parameters.  
+A binding-specific attribute is applied to each binding parameter supplied to the entry point method. The attribute takes the binding information as parameters. In the previous example, the first parameter has a **QueueTrigger** attribute applied, indicating queue triggered function. The queue name and connection string setting name are passed as parameters to the **QueueTrigger** attribute.
```

Comment:
Call out where these parameters are being passed into (the `QueueTrigger` as oppose to the `Run` method)

Reply:
@andresnava Good clarification, but the paragraph should stay indented.  That clarifies that it goes with the code, which is itself indented to indicate it goes with this particular step.